src/components/icon/index.jsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
-import './index.scss'
 import PropTypes from 'prop-types'
+
+import { icon as iconClassName } from './index.css'
```

Comment:
This is undefined  😢

Reply:
when you use it like this it works just fine.

```
import css from './index.css'

...

    <svg className={css.icon} viewBox={`${icon.viewBox}`}>
      <use xlinkHref={`#${icon.id}`} />
    </svg>
```

Reply:
It's weird. I read the Babel output, it was correct. I will take a look